firestore/src/include/firebase/firestore.h

```diff
@@ -264,8 +264,16 @@ class Firestore {
    * transaction is valid as long as the Future has not yet completed.)
    *
    * @return A Future that will be resolved when the transaction finishes.
+   *
+   * @note This method is only available when using the STLPort C++ runtime
```

Comment:
Please remove the `@note` part. It would be true if this method was `#ifdef`ed, but we didn't do it, and adding the `#ifdef` now would be essentially equivalent to removing it.

Reply:
Done